cdc_replication_test.py

```diff
@@ -41,11 +41,15 @@ def write_cql_result(res, path: str):
         os.fsync(file.fileno())
 
 
+def is_gemini_test(env):
+    return 'STRESSOR_SCT_' in env and env['STRESSOR_SCT_'] == 'gemini'
+
```

Comment:
Better use SCT_STRESSOR, it will automatically passed to ClusterTester.params dict